BBus/BBus/BusRoute/View/BusStationTableViewCell.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  BusStationTableViewCell.swift
+//  BBus
+//
+//  Created by 최수정 on 2021/11/02.
+//
+
+import UIKit
+
+class BusStationTableViewCell: UITableViewCell {
+    
+    enum BusRootCenterImageType {
+        case wayPoint, uturn
+    }
+
+    static let reusableID = "BusStationTableViewCell"
+    static let cellHeight: CGFloat = 80
+    
+    var titleLeadingOffset: CGFloat { return 107 }
+    var lineTrailingOffset: CGFloat { return -20 }
+    
+    private lazy var beforeLineView = UIView()
+    private lazy var afterLineView = UIView()
```

Comment:
어떤 역할을 하는 라인인지 모호할 수 있을 것 같아요. 혹시 이런 네이밍은 어떤가요?
```suggestion
    private lazy var beforeCongestionLineView = UIView()
    private lazy var afterCongestionLineView = UIView()
```